resharper/src/resharper-unity/Psi/ShaderLab/Parsing/ShaderLab.lex

```diff
@@ -100,6 +97,8 @@ PP_DIGITS={DECIMAL_DIGIT}(({WHITESPACE})*{DECIMAL_DIGIT})*
 <YYINITIAL,PARENS>          ","                 { return ShaderLabTokenType.COMMA; }
 <YYINITIAL>     "."                             { return ShaderLabTokenType.DOT; }
 <YYINITIAL>     "+"                             { return ShaderLabTokenType.PLUS; }
+<YYINITIAL>     "-"                             { return ShaderLabTokenType.MINUS; }
+<YYINITIAL>     "+-"                            { return ShaderLabTokenType.PLUS_MINUS; }
```

Comment:
When can this be used (just curious)? :)

Reply:
It's a [legacy operator](https://github.com/JetBrains/resharper-unity/blob/shaderlab-parser-improvements/resharper/src/resharper-unity/Psi/ShaderLab/Parsing/ShaderLab.psi#L1855), used as part of the `Combine` calculation, same as `dot3` and `dot3rgba`. It needs to be flagged as an error with an inspection.